synapse/server_notices/resource_limits_server_notices.py

```diff
@@ -80,7 +80,12 @@ def maybe_send_server_notice_to_user(self, user_id):
             # In practice, not sure we can ever get here
             return
 
-        room_id = yield self._server_notices_manager.get_notice_room_for_user(user_id)
+        # Retrieve or create a server notices room for this user. This function is
+        # cached, so if the user hasn't been invited to a room previously created yet,
+        # a new one won't be created.
```

Comment:
I don't understand this sentence?

Reply:
Probably not a helpful comment anyway, so I'll just remove it.